lib/openxml/docx/elements/abstract_number.rb

```diff
@@ -0,0 +1,26 @@
+module OpenXml
+  module Docx
+    module Elements
+      class AbstractNumber < OpenXml::Docx::Element
+        include HasChildren, HasProperties
+        tag :abstractNum
+
+        def initialize(id)
+          super()
+          self.id = id
+        end
+
+        with_namespace :w do
+          attribute :id, expects: :integer, displays_as: :abstractNumId, required: true
+        end
+
+        # value_property :nsid
+        value_property :multi_level_type
```

Comment:
Without your custom override of `property_xml` as you have elsewhere, this will be dumped by default in a `<w:abstractNumPr/>` tag, which is probably not what you want. 🙂

Reply:
As a side note, `openxml_package`contains a new module named `ContainsProperties`, which addresses this issue; however, `openxml_docx` hasn't yet been refactored to take advantage of it throughout, though perhaps you could include `OpenXml::ContainsProperties` here (and elsewhere that you want this direct-child properties behavior), and I believe it _should_ work without you having to override `property_xml` each time...

Reply:
I may do that in a future version - for now, I'm just going to fix it the "easy" way. (Although I'll likely switch to `OpenXml::ContainsProperties` after implementing tests to make sure the XML is generated correctly.)